rc/extra/elixir.kak

```diff
@@ -0,0 +1,85 @@
+# http://elixir-lang.org
+# ----------------------
+
+# Detection
+# ‾‾‾‾‾‾‾‾‾
+
+hook global BufCreate .*[.](ex|exs) %{
+    set buffer filetype elixir
+}
+
+# Faces
+# -----
+
+face ElixirAtom yellow
+face ElixirUnderscore white+d
+face ElixirAttribute green+d
+face ElixirSpecialKeyword blue+b
+face ElixirEnv red+b
+face ElixirOperator rgb:a8a8a8
+face ElixirInterpolation rgb:00d7ff
```

Comment:
Using custom faces here means colorschemes wont be redefining these. I think it will be better to use the generic ones (see for example `colors/default.kak`)

If you think we really need to introduce new generic ones, we can discuss that either here or on IRC, but I dont think adding filetype specific faces is the way to go.